Sources/Widgets/Widgets3D/LineWidget/behavior.js

```diff
@@ -109,6 +176,69 @@ export default function widgetBehavior(publicAPI, model) {
     }
   };
 
+  publicAPI.toggleHandleVisibility = (bv) => {
```

Comment:
please explain what this function is supposed to do. I feel it can be written in 4 lines.

Reply:
Related to shapeMixin upgrade